README.md

```diff
@@ -0,0 +1,40 @@
+# XMTP React Playground
+
+**XMTP React Playground demonstrates core and advanced capabilities of the XMTP JS client SDK, aiming to showcase effective and innovative ways of building with XMTP with limited design.**
+
+The XMTP React Playground is built with React and the [XMTP client SDK for JavaScript](https://github.com/xmtp/xmtp-js) (`xmtp-js`).
+
+This app is maintained by [XMTP Labs](https://xmtplabs.com) and distributed under [MIT License](./LICENSE) for learning about and developing apps built with XMTP (Extensible Message Transport Protocol), the open protocol and network for secure web3 messaging.
+
+You are free to customize and deploy the app.
+
+This app has not undergone a formal security audit.
+
+> **Note**  
+> You might also be interested in our Vue Plaground -- Coming Soon 
+
+## Get started
+
+### Install the package
+
+```bash
+npm install
+```
+
+### Run the development server
+
+```bash
+npm run dev
+```
+
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the app.
+
```

Comment:
@nakajima or @rygine can one of you fill this out? Not sure if this is the steps for running this locally.

Reply:
yep, that's the command for local dev